README.pg_sphere

```diff
@@ -13,7 +13,20 @@ This is an R-Tree implementation using GiST for spherical objects like
 spherical points and spherical circles with useful functions and operators.
 
 NOTICE:
-     This version will work only with PostgreSQL version 9.6 and above.
+     This version will work only with PostgreSQL version 10 and above.
+
+VERSIONING:
+
+Stable versions are marked with tags containing version numbers. Each stable
```

Comment:
Change "Stable versions are marked with tags containing version numbers" to "Stable versions are marked with tags containing version numbers in the GitHub repository at https://github.com/postgrespro/pgsphere/".

Reason: Someone might download a tarball release or fork the repository and not have the context of where these tags can be found.